test/test_dataloader.py

```diff
@@ -506,17 +585,22 @@ def test_error_workers(self):
         self._test_error(DataLoader(ErrorDataset(41), batch_size=2, shuffle=True, num_workers=4))
 
     @unittest.skipIf(IS_WINDOWS, "FIXME: stuck test")
-    @unittest.skipIf(not TEST_CUDA, "CUDA unavailable")
     def test_partial_workers(self):
         r"""Check that workers exit even if the iterator is not exhausted."""
-        for pin_memory in (True, False):
+        if TEST_CUDA:
+            pin_memory_configs = (True, False)
```

Comment:
What does that mean? 😮

Reply:
It means that we don't test `pin_memory=True` when there is no CUDA.

Reply:
Omg I misunderstood the code